NBitcoin/Secp256k1/ECMultContext.cs

```diff
@@ -222,6 +222,34 @@ public GEJ Mult(in GEJ a, in Scalar na, in Scalar ng)
 				return ECMultiply(state, @as, nas, ng, 1);
 			}
 		}
+
+#if SECP256K1_LIB
+		public
+#else
+		internal
+#endif
+
+		GEJ Mult(in Span<GEJ> @as, in Span<Scalar> nas)
+		{
+			var n = @as.Length;
+			unsafe
+			{
+				GEJ* prej = stackalloc GEJ[n * ArraySize_A];
+				FE* zr = stackalloc FE[n * ArraySize_A];
+				GE* pre_a = stackalloc GE[n * ArraySize_A];
+				GE* pre_a_lam = stackalloc GE[n * ArraySize_A];
+				StraussPointState* ps = stackalloc StraussPointState[n];
```

Comment:
The stack space is limited, you should not allocate unless you precisely know that it can't be more than a bounded size, and small enough. (say less than 500 bytes)

But here we do not control `n`.
I don't think just using heap allocated pointers (either managed or unmanaged), will have big performance impact.

Reply:
Yes, you're right. I understand i have to allocate those arrays in the heap,  am i right? Please take a look at my last commit.